server/index.js

```diff
@@ -26,11 +32,16 @@ const { staticMiddlewares } = require("./middlewares");
 const app = express();
 app.use(express.json());
 
+app.use((req, res, next) => {
+  // If we have a fundamental redirect mimic out Lambda@Edge and redirect.
+  const { url: fundamentalRedirectUrl, status } = resolveFundamental(req.url);
+  if (fundamentalRedirectUrl && status) {
+    return res.redirect(status, fundamentalRedirectUrl);
+  }
+  return next();
+});
 app.use(staticMiddlewares);
 
-app.use(express.static(STATIC_ROOT));
-
-app.use(express.json());
```

Comment:
serve static is included in `staticMiddlewares` / `express.json()` has already been added.

Reply:
Thanks for noting these points. I was wondering the same, why we included each of `express.json()` and `express.static(STATIC_ROOT)` twice in the express middleware stack.